ideascube/library/migrations/0010_section_type.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.1 on 2016-10-28 08:32
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+int_to_name = {1: 'digital',
+               2: 'children-cartoons',
+               3: 'children-novels',
+               10: 'children-poetry',
+               11: 'children-theatre',
+               4: 'children-documentary',
+               5: 'children-comics',
+               6: 'adults-novels',
+               7: 'adults-documentary',
+               8: 'adults-comics',
+               12: 'adults-poetry',
+               13: 'adults-theatre',
+               9: 'game',
+               99: 'OTHER'}
+
+def migrate_data(app, schema_editor):
+    Book = app.get_model('library', 'Book')
+    db_alias = schema_editor.connection.alias
+    for book in Book.objects.using(db_alias).all():
+        book.section_named = int_to_name[book.section]
+        book.save()
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('library', '0009_auto_20161027_0801'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='book',
+            name='section_named',
+            field=models.CharField(choices=[('digital', 'digital'), ('children-cartoons', 'children - cartoons'), ('children-novels', 'children - novels'), ('children-poetry', 'children - poetry'), ('children-theatre', 'children - theatre'), ('children-documentary', 'children - documentary'), ('children-comics', 'children - comics'), ('adults-novels', 'adults - novels'), ('adults-poetry', 'adults - poetry'), ('adults-theatre', 'adults - theatre'), ('adults-documentary', 'adults - documentary'), ('adults-comics', 'adults - comics'), ('game', 'game'), ('OTHER', 'other')], default='OTHER', max_length=50, verbose_name='section'),
```

Comment:
Hum, there should be no default at all for section (there was no default before)
I add a default here cause we need something to put in this field during the migration.

So, a `preserve_default=False` is missing in the `AddField` call and I will remove the default in the model definition.